Example/PinwheelSDK.xcodeproj/project.pbxproj

```diff
@@ -13,7 +13,10 @@
 		607FACDD1AFB9204008FA782 /* Images.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 607FACDC1AFB9204008FA782 /* Images.xcassets */; };
```

Comment:
Should this file have been changed/checked in? Not familiar with iOS so just a genuine question.